doc/lua_api.txt

```diff
@@ -9299,20 +9299,31 @@ Chat command definition
 
 Used by `minetest.register_chatcommand`.
 
+Specifies the function to be called and the privileges required when a player
+issues the command.  A help message that is the concatenation of the params and
+description fields is shown when the "/help" chatcommand is issued with the
+command name as its argument, 
```

Comment:
overly specific, `/help all` shows these too for example
just be more general and say it appears in `/help`?

Reply:
Okay.